corehq/apps/smsforms/app.py

```diff
@@ -56,9 +61,20 @@ def start_session(domain, contact, app, module, form, case_id=None, yield_respon
         }
     
     if app and form:
```

Comment:
kookoo tests pass. corehq.apps.sms.tests.inbound_handlers.KeywordTestCase.test_all_inbound fails at line 124 with "AssertionError: u'arm_a' != 'arm_b'" in both this branch and on master. Checking it out.